teams/fuchsia.toml

```diff
@@ -6,6 +6,6 @@ leads = []
 members = [
     "tmandry",
     "djkoloski",
-    "ComputerDruid",
+    "erickt",
```

Comment:
Doesn't this require changing files in `people/` too?

Reply:
Indeed @tmandry CI is failing.